src/templates/kanban-template/kanban-template.stories.tsx

```diff
@@ -1,30 +1,48 @@
 import type { Meta, StoryObj } from "@storybook/react";
 import * as React from "react";
-import { useEffect, useState } from "react";
 
 import { Badge, Outline } from "../../components";
-import { AttributeData } from "../../lib";
-import { KanbanTemplate } from "./kanban";
+import { KanbanTemplate } from "./kanban-template";
 
 const meta: Meta<typeof KanbanTemplate> = {
   title: "Templates/Kanban",
   component: KanbanTemplate,
   argTypes: { onClick: { action: "onClick" } },
 };
 
+const generateComponentList = (count: number) => {
+  const randomLorenIpsum = [
+    "Lorem ipsum dolor sit amet, consectetur adipiscing elit.",
+    "Sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.",
+    "Ut enim ad minim veniam, quis nostrud exercitation ullamco.",
+    "Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.",
+    "Excepteur sint occaecat cupidatat non proident, sunt in.",
+  ];
+  return Array.from({ length: count }, (_, i) => (
+    <div key={i} style={{ display: "flex", flexDirection: "column" }}>
+      <span>20 days</span>
+      <span>{randomLorenIpsum[i % randomLorenIpsum.length]}</span>
+      <span>Some more data</span>
+    </div>
+  ));
+};
```

Comment:
This is the same as in the kanban.stories.tsx right? If yes, maybe it can be refactored?

Reply:
Noted